css/modals/invite/_add-people.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * Styles errors and links in the AddPeopleDialog.
+ */
+.modal-dialog-form {
+    .add-people-form-wrap {
+
+        .error {
+            padding-left: 5px;
+
+            a {
+                padding-left: 5px;
+            }
+        }
+    }
+}
+
+/**
+ * Styles the loading element in the MultiSelectAutocomplete.
+ */
+.autocomplete-loading {
+    -webkit-box-pack: center;
```

Comment:
Can justify-content be used instead of box-pack? MDN shows a replacement warning for box-pack.